features/step_definitions/sample_file_steps.rb

```diff
@@ -53,6 +53,10 @@ def a
   contents = <<-EOS.strip_heredoc
     # clean class for testing purposes
     class Clean
+      def initialize(sub)
+        @sub = sub
+      end
+
```

Comment:
I think its ok to just reference the existing files in our sample steps. Its not like our specs are going anywhere :)